packages/playground-examples/copy/ja/TypeScript/Primitives/Union and Intersection Types.ts

```diff
@@ -0,0 +1,83 @@
+// 型共用体は、いくつかの型である可能性がある
+// オブジェクトを宣言する一つの方法です。
+
+type StringOrNumber = string | number;
+type ProcessStates = "open" | "closed";
+type OddNumbersUnderTen = 1 | 3 | 5 | 7 | 9;
+type AMessyUnion = "hello" | 156 | { error: true };
+
+// "open" | "closed"を利用すること、string型を利用すること、
```

Comment:
このexampleがunion type初見者向けなので、ここのコメントに `|` を持ってきてしまうのは避けたほうが良いとおもいます（原文でも `"open" and "closed"` ですし）。

```suggestion
// string型ではなく、"open"や"closed"の利用が初めてであれば、
```